UefiPayloadPkg/SPI/pci_type.h

```diff
@@ -0,0 +1,30 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
Use PciLib from EDK2 and delete this file

Reply:
@IgorBagnucki as above

Reply:
Done